pkgs/top-level/all-packages.nix

```diff
@@ -20569,6 +20571,33 @@ with pkgs;
 
   megam = callPackage ../applications/science/misc/megam { };
 
+  toto = (pythonPackages.toPythonApplication (
+    pythonPackages.libxml2));
+  # (pkgs.mininet.override{ inherit python; pythonSupport = true;}).py
+  # );
+  # ((mininet-python).py);
+
+  mn-wrapped = pkgs.runCommand "mnexec-wrapper"
+    { buildInputs = [ pkgs.makeWrapper ]; }
+    ''
+      # This wrapper prevents from polluting PATH
+      makeWrapper ${mn}/bin/mn \
+        $out/bin/mn \
+        --prefix PATH : "${pkgs.telnet}/bin/telnet"
+    '';
+
+  # # with pythonPackages; toPythonApplication
+  # mn = let
+  #     mnApp = with pythonPackages; toPythonApplication mininet-python;
+  #   in  pkgs.runCommand "mnexec-wrapper"
+  #   { buildInputs = [ pkgs.makeWrapper ]; }
+  #   ''
+  #     # This wrapper prevents from polluting PATH
+  #     makeWrapper ${mnApp}/bin/mn \
+  #       $out/bin/mn \
+  #       --prefix PATH : "${pkgs.telnet}/bin/telnet"
+  #   '';
```

Comment:
Those lines should be removed from all-packages.nix. mn can be wrapped in the package.